nix/README.md

```diff
@@ -0,0 +1,63 @@
+# Nix-based kubernetes test environment
+
+All operations are based on having the `nix` package manager available. It can be easily be installed using the quickstart instructions found at https://nixos.org/download.html#nix-quick-install, which are just
+
+```console
+$ curl -L https://nixos.org/nix/install | sh
+```
+
+## Building a docker image of a node
+
+To build your current source into a container image, enter a nix-shell in the same folder as this `README.md`, then run `build-node.sh`.
+
+```console
+$ nix-shell
+$ ./build-node.sh
+[...]
+Created new docker image casper-node:f2b9cd7a-dirty.
```

Comment:
Using a `nix-shell` here doesn't really follow best practices.  Is it a big lift to have `nix-build` create the local docker in `result`?